src/routes/api/cards/+server.ts

```
@@ -0,0 +1,30 @@
+import { json } from '@sveltejs/kit';
+import type { RequestHandler } from './$types';
+import { getCards, searchCards } from '$lib/db';
+
+export const GET: RequestHandler = async ({ url }) => {
+  const query = url.searchParams.get('q');
+  const limit = parseInt(url.searchParams.get('limit') || '50');
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)